coops-ui/src/components/AnnualReport/Directors.vue

```diff
@@ -598,7 +609,21 @@ export default class Directors extends Mixins(DateMixin, ExternalMixin, EntityFi
   private get directorFirstNameRules (): Array<Function> {
     return [
       v => !!v || 'A first name is required',
-      v => !/\s/g.test(v) || 'Invalid spaces'
+      v => !/^\s/g.test(v) || 'Invalid spaces', // leading spaces
+      v => !/\s$/g.test(v) || 'Invalid spaces', // trailing spaces
```

Comment:
The original space rule prevented leading and trailing spaces. Were these implemented to allow multiple names in a single name input field?

Reply:
The original space rule prevented a space anywhere in the field. These rules were implemented to allow compound names. For example, my previous PO's first name is "Ann Marie".

Reply:
Same as above, good job 👍